src/components/implementation/tests/unit_pingshmem/Makefile

```diff
@@ -0,0 +1,18 @@
+# Required variables used to drive the compilation process. It is OK
+# for many of these to be empty.
+#
+# The set of interfaces that this component exports for use by other
+# components. This is a list of the interface names.
+INTERFACE_EXPORTS =
+# The interfaces this component is dependent on for compilation (this
+# is a list of directory names in interface/)
+INTERFACE_DEPENDENCIES = init capmgr pongshmem
+# The library dependencies this component is reliant on for
+# compilation/linking (this is a list of directory names in lib/)
+LIBRARY_DEPENDENCIES = kernel ps
```

Comment:
Are both of these required? I'd be surprised if `ps` was required. Maybe the same for `kernel`?

If thye aren't, maybe we should update the normal `ping` and `pong`.